dds/DCPS/LocalObject.h

```diff
@@ -36,6 +36,9 @@ class LocalObjectBase
   {
     RcObject::_remove_ref();
   }
+  virtual CORBA::ULong _refcount_value() const {
```

Comment:
```suggestion
  virtual CORBA::ULong _refcount_value() const
  {
```

Reply:
Sorry for that! I see another like it lower down the file but am sure I just carelessly fell into the habit rather than trying to follow suit. Same line brace is/was the style used on Borland product code. Thanks.

Reply:
Not a problem.  Since the current code is a mix of different styles we realize it's not always clear.  Thanks for your contribution.